scripts/hash-exec-copy.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -e
+
+[[ "$1" =~ https://raw.githubusercontent.com/makerdao/community/* ]] || { echo "Please provide the correct exec copy link to hash (e.g. url=https://raw.githubusercontent.com/makerdao/community/...)"; exit 1; }
```

Comment:
I like this validation, was about to propose it before I saw this line (as a common mistake is to use the non raw permalink path like https://github.com/makerdao/community/blob/master/governance/votes/Executive vote - October 5%2C 2022.md)

Consider maybe even doing `https://raw.githubusercontent.com/makerdao/community/*/governance/votes/*.md`

Reply:
done in f53f8b9